### DISCUSSION TITLE
Need a Way to Distinguish Nested `group-hover` for Elements

### DISCUSSION THREAD

**Discussion by Weswwn** [tailwindlabs/tailwindcss#7075] [Ideas]
Hi Everyone, 

Currently, there is no way of distinguishing nested `group-hovers` for nested elements that require different hover needs. I'm wondering if there's anything in the pipeline that will address this? 

Looking around online, it seems others have this problem as well and there is even an npm package dedicated to this: 
https://www.npmjs.com/package/tailwindcss-nested-groups

Would love to chat about this. 

Thanks.

**Comment by adamwathan** [comment]
I've had it in my idea list for awhile to explore some sort of named group feature, like `group[1]-hover` sort of thing. Something I'm hoping to play with in the next month or so for sure.

**Reply by Weswwn** [reply]
Sweet! Looking forward to it.

**Comment by reinink** [comment]
Hey! Good news — we just added nested group and multiple peer support using variant modifiers in Tailwind CSS v3.2 🥳

You can read about it in our [announcement blog post](https://tailwindcss.com/blog/tailwindcss-v3-2#nested-group-and-multiple-peer-support-using-variant-modifiers), but for quick reference here’s what it looks like:

```html
<div class="group/sidebar ...">
  <!-- ... -->
  <div class="group/navitem ...">
    <a
      href="#"
      class="opacity-50 group-hover/sidebar:opacity-75 group-hover/navitem:bg-black/75"
    >
      <!-- ... -->
    </a>
  </div>
  <!-- ... -->
</div>
```

This lets you give each group a clear name that makes sense for that context on the fly, and Tailwind will generate the necessary CSS to make it work.

**Reply by Weswwn** [reply]
Fantastic, thanks for the update!
